recipes/gnutls/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "gnutls" %}
+{% set version = "3.5.15" %}
+{% set sha256 = "046081108b8b1fe455a13a4c5a4eaa0368e185b678f1670fe09a11a2d7ecfad5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: ftp://ftp.gnutls.org/gcrypt/{{ name }}/v3.5/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - pkgconfig.patch
+
+
+build:
+  number: 0
+  skip: True  # [win]  
+
+requirements:
+  build:
+    - gmp 6.1.*
+    - nettle
+    - zlib 1.2.8 
+    - toolchain3
+    - libidn11
+    - pkg-config  # [osx]
+    - openssl 1.0.*
```

Comment:
Is `openssl` getting linked in statically?

Reply:
I'm trying to find out I don't know... I have added it because without it the osx build would fail... And I also added the gnu-tls-openssl compatibility but I don't think it's compulsory.

Reply:
I am not really sure how to answer this question, I have looked into the configure, Makefile and the logs from the building process and I don't really know... However openssl.so is not needed by the package libraries during the tests so I imagine it is linked statically ? Is there something wrong with it ?

Reply:
Ok i think I finally got it, the file included in gnutls, in extra/includes/gnutls/openssl.h provides the definition needed for building the gnutls-openssl compatibility. I think this is not optimal and anyway this compatiblity should not be used according to  the developers, I will try to understand what goes wrong when I don't include openssl as requirement in osx instead.